docker-compose.dev.yml

```diff
@@ -2,15 +2,15 @@ version: '3'
 
 services:
   webserver:
-    image: fnagel/docker-typo3-webserver-dev:${DOCKER_PHP_TAG}
+    image: ${DOCKER_BASE_IMAGE}:${DOCKER_PHP_TAG}
```

Comment:
The idea was to create two webserver images. One for production and one for development. With using the `DOCKER_BASE_IMAGE` variable, one can control on which image the demo or project docker compose file should use.

With this change it's no longer possible to create the dev image.

What about introducing a variable called `DOCKER_IMAGE_NAME` or similar and use it as a replacement for all `fnagel/docker-typo3-webserver` occurrences. This way, one could control how the image is named but we keep the prod / dev image creation working.